scripts/download_test_resources.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import sys
+import traceback
+from pathlib import Path
+
+from quilt3 import Package
+
+###############################################################################
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)4s: %(module)s:%(lineno)4s %(asctime)s] %(message)s",
+)
+log = logging.getLogger(__name__)
+
+###############################################################################
+# Args
+
+
+class Args(argparse.Namespace):
+    def __init__(self):
+        self.__parse()
+
+    def __parse(self):
+        # Setup parser
+        p = argparse.ArgumentParser(
+            prog="download_test_resources",
+            description=(
+                "Download files used for testing this project. This will download "
+                "all the required test resources and place them in the `tests/resources` "
+                "directory."
+            ),
+        )
+
+        # Arguments
+        p.add_argument(
+            "--top-hash",
+            # Generated package hash from upload_test_resources
+            default="89b6219c95f10b3a0c55935792e8da84af4d07080d05a73f65e5feb1896c3c29",
+            help=("A specific version of the package to retrieve."),
+        )
+        p.add_argument(
+            "--debug",
+            action="store_true",
+            help="Show traceback if the script were to fail.",
+        )
+
+        # Parse
+        p.parse_args(namespace=self)
+
+
+###############################################################################
+# Build package
+
+
+def download_test_resources(args: Args):
+    # Try running the download pipeline
+    try:
+        # Get test resources dir
+        resources_dir = (
+            Path(__file__).parent.parent / "aicsimageio" / "tests" / "resources"
```

Comment:
```suggestion
            Path(__file__).parent / "tests" / "resources"
```

Reply:
Nope. This file lives in `scripts` so it needs to go up two levels from current file. First parent is `scripts` itself then second parent is `aicsimageio` repo directory, then down one to `aicsimageio` python package directory. :upside_down_face:

Reply:
Ah, then perhaps this:
```suggestion
            Path(__file__).parents[1] / "aicsimageio" / "tests" / "resources"
```

Reply:
Ehhhh not the biggest fan to be honest. Will wait for others to comment.